src/assets/i18n.json

```diff
@@ -0,0 +1,162 @@
+        },
+        "exp": {
+          "name": "Expert",
+          "abbreviation": "Exp"
+        },
+        "cha": {
+          "name": "Challenge",
+          "abbreviation": "Cha"
+        }
+      },
+      "extreme": {
+        "bas": {
+          "name": "Light",
+          "abbreviation": "Lgt"
+        },
+        "dif": {
+          "name": "Standard",
+          "abbreviation": "Std"
+        },
+        "exp": {
+          "name": "Heavy",
+          "abbreviation": "Hvy"
+        }
+      }
+    },
+    "draw": "Draw!",
+    "confirmClose": "Are you sure you want to leave?",
+    "updateReady": "Update available, refresh for the freshest code around!",
+    "credits": "Credits",
+    "lastUpdate": "Songs up to date as of {{date}}.",
+    "contact": {
+      "facebook": "Contact me on Facebook Messenger",
+      "twitter": "Contact me on Twitter",
+      "github": "Contribute on Github"
+    }
+  },
+  "ja": {
+    "dataSource": "DDRのバージョン",
+    "chartCount": "ドロー数",
+    "difficultyLevel": "レベル",
+    "upperBound": "上限",
+    "lowerBound": "下限",
+    "useWeightedDistributions": "レベル別に確率を設定する",
+    "style": "スタイル",
+    "single": "Single",
+    "double": "Double",
+    "difficulties": "難易度",
+    "weights": {
+      "explanation": "各レベルごとに数字で出やすさの重みを付ける",
+      "check": {
+        "title": "「17に10%が設定されている状態で17が2つ以上ドローされる」というような非常にまれな結果を除外",
+        "label": "厳密に設定を反映する"
+      }
+    },
+    "include": "特殊な譜面",
+    "controls": {
+      "show": "元に戻す",
+      "hide": "最小化する",
+      "invalid": "対象譜面がありません！",
+      "unlock": "未解禁曲を含める",
+      "tempUnlock": "現在解禁できない曲を含める",
+      "goldExclusive": "A20筐体限定曲を含める",
+      "extraExclusive": "EXTRA EXCLUSIVE曲を含める",
+      "usLocked": "日本限定曲を含める",
+      "removed": "削除曲を含める"
+    },
+    "shockArrows": "Shock Arrows",
+    "difficulty": {
+      "ace": {
+        "beg": {
+          "name": "Beginner",
+          "abbreviation": "Beg"
+        },
+        "bas": {
+          "name": "Basic",
+          "abbreviation": "Bas"
+        },
+        "dif": {
+          "name": "Difficult",
+          "abbreviation": "Dif"
+        },
+        "exp": {
+          "name": "Expert",
+          "abbreviation": "Exp"
+        },
+        "cha": {
+          "name": "Challenge",
+          "abbreviation": "鬼"
+        }
+      },
+      "extreme": {
+        "bas": {
+          "name": "Light",
+          "abbreviation": "楽"
+        },
+        "dif": {
+          "name": "Standard",
+          "abbreviation": "踊"
+        },
+        "exp": {
+          "name": "Heavy",
+          "abbreviation": "激"
+        }
+      }
+    },
+    "draw": "ドロー！",
+    "confirmClose": "Are you sure you want to leave?",
+    "updateReady": "Update available, refresh for the freshest code around!",
+    "credits": "Credits",
+    "lastUpdate": "{{date}}現在の最新曲。",
+    "contact": {
+      "facebook": "Facebook Messengerで私に連絡してください",
+      "twitter": "Twitterで私に連絡してください",
+      "github": "Githubに貢献する"
+    }
```

Comment:
Added a bunch of new strings here. I used Google to translate some of them. I don't know if it even makes sense to have translations for every single one of these, but they're here just in case. Feel free to tweak everything here so it looks good. 😄